Integrations/integration-TruSTAR.yml

```diff
@@ -254,12 +276,24 @@ script:
 
     def get_report_details(report_id, id_type):
         response = ts.get_report_details(report_id, id_type)
-        report_details = response.to_dict(remove_nones=True)
-        if report_details['enclaveIds']:
-            report_details['enclaveIds'] = ', '.join(report_details['enclaveIds']) # Prettify list of enclave IDs
-        report_details['updated'] = normalize_time(report_details['updated'])
-        report_details['created'] = normalize_time(report_details['created'])
-        report_details['timeBegan'] = normalize_time(report_details['timeBegan'])
+        current_report_dict = response.to_dict(remove_nones=True)
+        report_details = collections.OrderedDict()
+        report_details['id'] = current_report_dict['id']
+        report_details['title'] = current_report_dict['title']
+        deep_link = "{server_url}/constellation/reports/{report_id}".format(server_url=SERVER,
+                                                                            report_id=current_report_dict['id'])
+        report_details['reportDeepLink'] = "[{}]({})".format(deep_link, deep_link)
```

Comment:
Our code convention is using `'text'` instead of `"text"`, please change accordingly.